Source/UI/BillingForm/View/BillingFormTextFieldView.swift

```diff
@@ -89,6 +83,17 @@ class BillingFormTextFieldView: UIView {
         updateErrorView(style: style)
     }
 
+    func refreshLayoutComponents() {
+        let handlesPhoneNumber = isPhoneNumberType()
```

Comment:
Is this naming inconsistency ? we are using isPhoneNumberType to return Bool with is prefix and storing in handlesPhoneNumber without a is prefix. Should we keep them consistent across.

Reply:
I don't want to confuse method and property. If `isPhoneNumberType` was also a property it would be ok, but as its a function I tried to name it differently and not change it. But happy to change as it might help make more sense.